gallery/gpsdrawerhelper.h

```diff
@@ -0,0 +1,76 @@
+#ifndef GPSDRAWERHELPER_H
+#define GPSDRAWERHELPER_H
+
+#include <QObject>
+#include <QString>
+#include "qgispoint.h"
+
+class  GpsDataHelper : public QObject
+{
+    Q_OBJECT
+
+    //Q_PROPERTY( PositionProvider* positionProvider READ positionProvider CONSTANT )
+    Q_PROPERTY( double latitude READ latitude CONSTANT )
+    Q_PROPERTY( double longitude READ longitude CONSTANT )
+    Q_PROPERTY( double verticalAccuracy READ verticalAccuracy CONSTANT )
+    Q_PROPERTY( double horizontalAccuracy READ horizontalAccuracy CONSTANT )
+    Q_PROPERTY( double altitude READ altitude CONSTANT )
+    Q_PROPERTY( double speed READ speed CONSTANT )
+    Q_PROPERTY( double x READ x CONSTANT )
+    Q_PROPERTY( double y READ y CONSTANT )
+    Q_PROPERTY( int satellitesUsed READ satellitesUsed CONSTANT )
+    Q_PROPERTY( int satellitesVisible READ satellitesVisible CONSTANT )
+    Q_PROPERTY( bool hasPosition READ hasPosition CONSTANT )
+    Q_PROPERTY( bool positionProvider READ positionProvider CONSTANT )
+    Q_PROPERTY( QString providerName READ providerName CONSTANT )
+    Q_PROPERTY( QString providerType READ providerType CONSTANT )
+    Q_PROPERTY( QString providerMessage READ providerMessage CONSTANT )
+    Q_PROPERTY( QString stateMessage READ stateMessage CONSTANT )
+    Q_PROPERTY( QString lastRead READ lastRead CONSTANT )
+    Q_PROPERTY( QgisPoint positionCoordinate READ positionCoordinate CONSTANT )
```

Comment:
this is not needed in GpsPanel - it reads directly x and y;... just remove this so you do not need QgsPoint too.